### ISSUE TITLE
Uncaught ReferenceError: h$isNode is not defined

### ISSUE THREAD

**Issue by spl** [ghcjs/ghcjs-jquery#9]
This is due to c5eeeafcf81c0d3237b8b9fcb98c4b3633a1297f and happens if `GHCJS_BROWSER` is defined. (I'm not really sure why nobody has reported this problem before. It could be something else about the way we're building our app, but I'm not really sure what that would be.)

Should the line be changed as follows?

```diff
-  if ( h$isNode ) return;
+  if (typeof h$isNode !== "undefined") return;
```

**Comment by adinapoli** [comment]
Ah, I have just stumbled into this myself while trying to fix #14 and as a consequence improve the shape of `example/Main.hs`.

The fact we are running a patched version of the jquery library surprise me a bit. Isn't there a way to ship the official, minified library?

**Comment by spl** [comment]
I haven't followed up on this, and I guess it's somewhat telling that nobody else has since I posted it over 3 months ago. We were using a patched `ghcjs-jquery` but stopped due to my failure at resolving #10. I only needed a few jQuery features, so I wrote the minimal necessary FFI myself.

**Comment by adinapoli** [comment]
@spl I have opened a PR 2 days ago addressing this as well: https://github.com/ghcjs/ghcjs-jquery/pull/15

Unfortunately nobody reviewed it yet..

**Comment by bergey** [comment]
closed by #15
